src/js/components/DataFilter/DataFilter.js

```diff
@@ -116,7 +128,13 @@ export const DataFilter = ({
           label
           min={range[0]}
           max={range[1]}
-          step={(range[1] - range[0]) / 20}
+          step={
+            // from `range` on DataFilter
+            rangeProp?.step ||
+            // from range in Data `properties`
+            properties?.[property]?.range?.step ||
+            (range[1] - range[0]) / 20
```

Comment:
Where does "20" come from? What is the rationalization behind that value?

Reply:
I think it was just a value that Eric chose as a consistent number of steps for a range. Not sure if there was any further rationalization beyond that.

Reply:
can we do something like `const defaultSteps = 20;` to use a variable to be self documenting?